### ISSUE TITLE
urllauncher is deprecated

### ISSUE THREAD

**Issue by jacopo1395** [loowcode/calendaroo#55]

**Comment by Kenwolf-byte** [comment]
Please i am facing the same issue. Due to this my flutter apk release version does not run

**Comment by jacopo1395** [comment]
I don't know how help you. We build an appbundle and it works.

**Comment by Kenwolf-byte** [comment]
> I don't know how help you. We build an appbundle and it works.

Yep mine is working now. All i did was to restart my machine and for some strange reason it worked. Though the deprecation still exists.
